src/app-designer-prereqs.rst

```diff
@@ -37,7 +37,7 @@ You must install the following software on your computer in order to use Applica
 
 .. warning::
 
-  Android Studio is not supported on the Windows Linux subsystem, so you will not be able to run Application Designer if you are using it. 
+  Android Studio is not supported on the Windows Subsystem for Linux (WSL). so you will not be able to run Application Designer if you are using it. 
```

Comment:
period after (WSL) should be a comma. Is the acronym needed? If not used elsewhere, can skip here.